parking_lot/shr_sexualhealth.txt

```diff
@@ -10,80 +10,80 @@ CodeSystem:     UCUM = http://unitsofmeasure.org
 
 EntryElement:	PregnancyIntention
 Concept:		TBD
-Based on:		QuestionAnswerRecorded
+Based on:		SocialHistoryObservation
 Description:	"Whether the subject or his/her partner has a plan or desire to either become pregnant or have a child in the next year/near future."
 Value:			CodeableConcept from YesNoUnknownVS
-				TopicCode is LNC#64631-5
+				Code is LNC#64631-5
 
 EntryElement:	SexualIdentity
 Concept:		TBD
-Based on:		QuestionAnswerRecorded
+Based on:		SocialHistoryObservation
 Description:	"Information on gender identity and sexual orientation."
-				TopicCode is TBD#TBD
+				Code is TBD#TBD
 				PanelMembers.EvaluationTopic
 includes 0..1 	GenderIdentity
 includes 0..1	SexualOrientation
 
 	EntryElement:	GenderIdentity
 	Concept:		MTH#C0017249
```

Comment:
NCI MTH C0017249 is "Mental Process" and not a good fit for Social History Observation.
Suggest MTH#C1287110 "Social and personal history finding" as a better fit.